js/panel_controller.js

```diff
@@ -19,11 +19,19 @@
 
     window.drawAttention = function() {
         if (inboxOpened && !inboxFocused) {
+            if( window.keepClear ) {
+                clearInterval( window.keepClear );
+                delete window.keepClear;
+            }
             extension.windows.drawAttention(inboxWindowId);
         }
     };
     window.clearAttention = function() {
-        extension.windows.clearAttention(inboxWindowId);
+        if( window.keepClear ) {
+            clearInterval( window.keepClear );
+            delete window.keepClear;
+        }
+        window.keepClear = setInterval( () => extension.windows.clearAttention(inboxWindowId), 100 );
```

Comment:
We're not really using arrow functions yet in this project, and they may even cause problems with some of the older Chrome builds we still support. Please change this to `function() {` and put the body of the function on its own line.